src/Aws/Lambda/Runtime.hs

```diff
@@ -17,16 +21,52 @@ import qualified Aws.Lambda.Runtime.Environment as Environment
 import qualified Aws.Lambda.Runtime.Error as Error
 import qualified Aws.Lambda.Runtime.Publish as Publish
 
+type Handler =
+  Event -> IO (Either Text Text)
+
+data Event =
+  Event {
+      memoryLimitInMb    :: !Int
+    , functionName       :: !Text
+    , functionVersion    :: !Text
+    , invokedFunctionArn :: !Text
+    , awsRequestId       :: !Text
+    , xrayTraceId        :: !Text
+    , logStreamName      :: !Text
+    , logGroupName       :: !Text
+    , deadlineMs         :: !Int
+    , handlerName        :: !Text
+    , clientContext      :: !Text
+    , event              :: !Text
```

Comment:
is `event` like the `request` argument and the rest of the fields are like the `lambdacontext` in go?

Reply:
just took every usable piece of info from their Context and LAmbdaOptions types and put them together without the [Char]